conda/core/solve.py

```diff
@@ -1193,19 +1195,26 @@ def _setup_state(self, **kwargs):
 
         return state
 
+    def _channel_urls(self):
+        channels = []
```

Comment:
Add a comment that this is to normalize the channel url handling between Conda's CLI based system and Mamba's str based system.